element_calcium_imaging/imaging_no_curation.py

```diff
@@ -971,15 +975,16 @@ class ActivityExtractionMethod(dj.Lookup):
 class Activity(dj.Computed):
     definition = """  # inferred neural activity from fluorescence trace - e.g. dff, spikes
     -> Fluorescence
-    -> ActivityExtractionMethod
+    -> ActivityExtractionParamSet
     """
 
     class Trace(dj.Part):
         definition = """  #
         -> master
         -> Fluorescence.Trace
+        activity_type: varchar(16) #  e.g. dF/F, calcium-event
```

Comment:
Please update `make` to insert this value.

Reply:
also a conflict resolution mistake - this is removed now